src/ServiceProviders/RepositoryProviders/Bitbucket/BitbucketProvider.php

```diff
@@ -199,7 +232,8 @@ public function addInteractions(InputInterface $input, OutputInterface $output)
         }
         $extras = $input->getOption('extra');
         if (!empty($extras['bitbucket-project'])) {
-            $this->project = $extras['bitbucket-project'];
+            $project_uuid = $this->getProjectUUID($extras['bitbucket-project'], $org);
+            $this->project = $project_uuid;
```

Comment:
Maybe skip assignment if $project_uuid is false, and fall-through to the interactive selection?

Reply:
Good suggestion! Updated